crowbar_framework/app/models/api/storage.rb

```diff
@@ -15,6 +15,43 @@
 #
 
 module Api
-  class Storages
+  class Storage < Tableless
+    def repocheck
+      {}.tap do |ret|
+        [:ceph, :system].each do |r|
+          ret[r] = {
+            available: false
+          }
+        end
+
+        # check for the system repos
+        ["sles12-sp2-pool", "sles12-sp2-updates"].each do |repo|
+          NodeObject.all.map(&:architecture).each do |arch|
+            repository = Crowbar::Repository.where(
+              platform: "suse-12.2",
```

Comment:
Similarly, hardcoding the platform everywhere... Maybe we should simply have a helper in crowbar-core that gives us the target platform when upgrading nodes?